.husky/pre-push

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npm run lint
```

Comment:
why do we need pre-push and pre-commit?

Reply:
I removed this file , we are going to keep with  pre-commit file